src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder_ListPatterns.cs

```diff
@@ -27,9 +28,7 @@ private Tests MakeTestsAndBindingsForListPattern(BoundDagTemp input, BoundListPa
             {
                 tests.Add(new Tests.One(new BoundDagTypeTest(list.Syntax, ErrorType(), input, hasErrors: true)));
             }
-            else if (list.HasSlice &&
-                     subpatterns.Length == 1 &&
-                     subpatterns[0] is BoundSlicePattern { Pattern: null })
+            else if (ListPatternHasOnlyEmptySlice(list))
```

Comment:
>ListPatternHasOnlyEmptySlice [](http://example.com/codeflow?start=21&length=28)

It looks like the extracted helper is not used anywhere else. Consider reverting changes to this file.